build_all.sh

```
@@ -0,0 +1,80 @@
+# Copyright 2021 Fuzz Introspector Authors
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)